interfaces/StarkNetBitcoinDepositor.ts

```
@@ -1,144 +1,750 @@
+// This ABI is from the StarkNet Bitcoin Depositor contract deployed on Sepolia
+// Source: tbtc-v2/cross-chain/starknet/deployments/sepolia/StarkNetBitcoinDepositor.json
 export const StarkNetBitcoinDepositorABI = [
   {
-    inputs: [
+    "inputs": [],
```

ESLint

[error] Replace `"inputs"` with `inputs`

(prettier/prettier)